tools/ptrace.c

```diff
@@ -142,9 +142,11 @@ read_string(char **buffer, int pid, Elf64_Addr addr)
     return 1;
   }
 
-  for (i = 0; i < len; i++)
-    if (read_byte((*buffer + i), pid, addr + i))
+  for (i = 0; i < len; i++) {
```

Comment:
I'd suggest moving this cosmetic change to a separate commit. Then, in the future, if someone is looking through the git history, they'll know that the change is cosmetic and will be able to ignore it.

Reply:
Easy, I might just drop it, I think I made it for a debugging statement